examples/wandb/wandb.ipynb

```diff
@@ -0,0 +1,330 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Weights & Biases"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Weights & Biases logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/wandb.png \"doc-image\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "This example shows how to use Weights & Biases to monitor the progress of model training on resource with a Dask Cluster in Saturn Cloud. This example will use PyTorch for image classification. It will use the Stanford Dogs dataset, and starting with a pre-trained version of Resnet50 will use transfer learning to make it perform better at dog image identification. In the (Weights & Biases (With Dask))[./wandb-dask.ipynb] example, the same example is used to show how to apply Weights & Biases when running a Dask cluster.\n",
```

Comment:
```suggestion
    "This example shows how to use Weights & Biases to monitor the progress of model training on resource with a Dask Cluster in Saturn Cloud. This example will use PyTorch for image classification. It will use the Stanford Dogs dataset, and starting with a pre-trained version of Resnet50 will use transfer learning to make it perform better at dog image identification. In the [Weights & Biases (With Dask)](./wandb-dask.ipynb) example, the same example is used to show how to apply Weights & Biases when running a Dask cluster.\n",
```